src/main/docker/cassandra-cluster.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
     timeline-cassandra:
-        image: cassandra:3.11
+        image: cassandra:3.9
```

Comment:
why you have reduced Cassandra version?

Reply:
this file was generated by new version of Jhipster

changes was reverted